src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -251,8 +257,14 @@ class CodeBlock private constructor(
 
       var p = 0
       while (p < format.length) {
+        if (format[p] in setOf('⇥', '⇤', '«', '»')) {
+          formatParts += format[p].toString()
+          p++
+          continue
+        }
+
         if (format[p] != '%') {
-          var nextP = format.indexOf('%', p + 1)
+          var nextP = format.indexOfAny(listOf("%", "«", "»", "⇥", "⇤"), p + 1)
```

Comment:
nit: Use the version that takes a `CharArray`

Reply:
👍Fixed that. I also found a bug in the midst of making that change in handling the new single character placeholders in `addNamed` 😄

Reply:
Ha, nice. Any new test cases that we should add?

Reply:
I ended up modifying the test `namedAndNoArgFormat` for this.